### ISSUE TITLE
To create a "Problems/F.A.Q." section

### ISSUE THREAD

**Issue by Vovcick** [vbonvin/IPT-public#12]
The proposal is to have some F.A.Q. section just after the problems list in our website, where we answer the most common questions about the problems and maybe even give some links that author prepared for this problem (this year it may be Popcorn or Magnetic fissioning of the droplet).
We try to publish only the most useful answers and links in order to make sure that the problem will be understood correctly by the teams

**Comment by vbonvin** [comment]
Good idea. We should also emphase to the IOC and participants that we expect them to come to us if they need clarifications.

**Comment by vbonvin** [comment]
I think the best way to do it is to integrate the FAQ directly between the problems in the[ IPT2017 page](http://2017.iptnet.info/problems/). I've added a small text right below the pdf link to encourage people to ask questions. 

If you can think of any questions that could be asked about any problems please let me know, so we can start filling the page with them.

**Comment by mhrz** [comment]
Vivien, take care, you've removed all the videos during your edit...

I'm currently correcting your mistake.

Maxime.

**Comment by mhrz** [comment]
Oh no, it's Joakim, sorry! I'm going to ask him the reason.

Maxime.

**Comment by vbonvin** [comment]
bumping the thread with this comment:

> If you can think of any questions that could be asked about any problems please let me know, so we can start filling the page with them.
